test/server/repositories/UrlRepository.test.ts

```diff
@@ -363,7 +360,7 @@ describe('UrlRepository', () => {
       WHERE query @@ (
   setweight(to_tsvector('english', urls."shortUrl"), 'A') ||
   setweight(to_tsvector('english', urls."description"), 'B')
-) AND urls.state = 'ACTIVE' AND urls."isSearchable"=true
+) AND urls.state = 'ACTIVE' AND urls.description != ''
```

Comment:
can i check why do we filter `urls.description != ''` again? i was under the impression that GoDirectory would make all URLs available.

Reply:
It would - GoDirectory will not touch those code paths that were exercised by that test suite - that's mostly for GoSearch, which we will nuke together with these tests.